openssl-dynamic/src/main/c/nativestaticallyreferencedjnimethods.c

```diff
@@ -0,0 +1,482 @@
+/*
```

Comment:
In fact, would it be better to call this something like `native_constants.c`? `nativestaticallyreferencedjnimethods` is a bit of a mouthful :).